src/components/subscriptionDialog.vue

```diff
@@ -0,0 +1,286 @@
+<template>
+  <v-dialog
+    id="subscription-dialog"
+    v-model="showSubscriptionDialog"
+    max-width="550"
+  >
+    <v-card style="display: flex; flex-direction: column; overflow: hidden;">
+      <div id="subscription-details">
+        <v-card-title class="headline">
+          Premium Subscription
+        </v-card-title>
+        <v-card-text data-cy="detail">
+          Subscribe to Ambianic Premium Services to enable advanced
+          features, such as instant email notification.
+        </v-card-text>
+        <div>
+          <h3
+            data-cy="price"
+            style="font-weight: 500; text-align: center;"
+          >
+            $5 Monthly Fee
```

Comment:
For now, we can use a hardcoded subscription fee. 

I would look into the Stripe docs on pulling details about a created product. However, I think it might be overkill to make network requests just for that. Any further thoughts?

Reply:
How about cashing the current user subscription fee in the Auth0 user profile. The user profile has to be fetched anyways on login.